test/queries.js

```diff
@@ -166,3 +281,9 @@ graphql(BigchainDBSchema, queryVoteList).then(result => {
 graphql(BigchainDBSchema, querySearch).then(result => {
     console.log(JSON.stringify(result, null, 2))
 })
+graphql(BigchainDBSchema, mutationTransaction).then(result => {
```

Comment:
are these used for the previous queries? or do you prepopulate your DB?

would be good if we use these as a prepopulation and we can do the query afterwards